hack/.ci/run-e2e-gke.sh

```diff
@@ -31,10 +31,23 @@ function kubectl_create {
 }
 
 function gather-artifacts {
-  ARTIFACTS_DIR="${ARTIFACTS}" timeout -v 50m ./hack/ci-gather-artifacts.sh
-  # Events are costly to gather, so we'll erase them for now. (e2e namespaces are collected separately)
-  kubectl delete --raw /api/v1/namespaces/default/events || true
-  kubectl delete --raw /api/v1/namespaces/kube-system/events || true
+  kubectl -n e2e run --restart=Never --image="${SO_IMAGE}" --labels='app=must-gather' --command=true must-gather -- bash -euExo pipefail -O inherit_errexit -c "function wait-for-artifacts { touch /tmp/done && until [[ -f '/tmp/exit' ]]; do sleep 1; done } && trap wait-for-artifacts EXIT && mkdir /tmp/artifacts && scylla-operator must-gather --all-resources --loglevel=2 --dest-dir=/tmp/artifacts"
+  kubectl -n e2e wait --for=condition=Ready pod/must-gather
+
+  # Setup artifacts transfer when finished and unblock the must-gather pod when done.
```

Comment:
Duplicate of the comment later in the file, is it intentional?

Reply:
it refers to different pods